landoui/dockerflow.py

```diff
@@ -16,8 +19,15 @@ def heartbeat():
     and return a 200 iff those services and the app itself are
     performing normally. Return a 5XX if something goes wrong.
     """
-    # TODO check backing services and update tests
-    return '', 200
+    try:
+        response = requests.get(
+            current_app.config['LANDO_API_URL'] + '/__lbheartbeat__'
+        )
+        response.raise_for_status()
+    except (requests.HTTPError, requests.ConnectionError):
```

Comment:
Thinking about it, you are right that we don't need to send the exception to Sentry.  I think it should be sent to the application log instead, as we do in lando-api.